src/lang/pt-BR/DotView.json

```diff
@@ -0,0 +1,4 @@
+{
+  "tableHeader": "Ponto Vista",
+  "tableSubHeader": "O status das tarefas de um conjunto conforme organizado por ponto de ciclo"
```

Comment:
Ponto Vista sounds like Point of View in Portuguese (ponto de vista). But "vista" doesn't really mean the same as in English like a display or visualization... I guess we would need to go with visualization here.

Maybe "Visualização por Pontos" for the `tableHeader` and for `tableSubHeader` "Os status das tarefas de uma suite organizados por pontos do ciclo."

Reply:
Excellent, thank you. The subtleties of translation 😁 Google translate has a tough job, for sure!